Tests/SonarScanner.MSBuild.Common.Test/ProjectInfo/ProjectInfoExtensionsTests.cs

```diff
@@ -136,4 +140,37 @@ public void GetProjectGuidAsString_WhenProjectInfoIsNull_ThrowsArgumentNullExcep
         // Assert
         action.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("projectInfo");
     }
+
+    [TestMethod]
+    public void GetAllAnalysisFilesTest()
+    {
+        var dir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
+        var filesToAnalyze = Path.Combine(dir, "FilesToAnalyze.txt");
+        var logger = new TestLogger();
+        var projectInfo = new ProjectInfo
+        {
+            AnalysisResults =
+            [
+                new AnalysisResult
+                {
+                    Id = "FilesToAnalyze",
```

Comment:
"FilesToAnalyze" is used in a lot of places in tests. Would be great to extract it to a constant.